run.zsh

```diff
@@ -9,14 +9,16 @@ fi
 
 typeset -A results
 spin=('/' '-' '\' '|')
-test_dir='/tmp/zsh-benchmark'
+test_dir="$(mktemp -d)-zsh-benchmark"
```

Comment:
When I was running this under different usernames this was giving me all kinds of trouble. It seems safer not to re-use the directory.